app/models/qernel/graph.rb

```diff
@@ -412,10 +411,6 @@ def graph_query
   end
 
   def initializer_inputs
-    decorated_inputs.sort_by { |input, _| [-input.priority, input.key] }
```

Comment:
Does this mean that init. inputs are no longer sorted by priority, or is it already done elsewhere?